src/main/java/models/Suit.java

```diff
@@ -6,6 +6,6 @@
  * Created by michaelhilton on 1/25/16.
  */
 public enum Suit  implements Serializable {
-    Hearts, Spades, Diamonds, Clubs
+    Hearts, Spades, Diamonds, Clubs, Cups, Swords, Golds
```

Comment:
Realistically we could have just kept the suits to clubs hearts diamonds and spades because the user wont really see the actual values, but this is probably better for the future.